tests/Integration/src/Listener/SitemapBlogListener.php

```diff
@@ -20,9 +20,8 @@
 use Presta\SitemapBundle\Sitemap\Url\UrlConcrete;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
-use Symfony\Component\Routing\RouterInterface;
 
-final class SitemapListener implements EventSubscriberInterface
+final class SitemapBlogListener implements EventSubscriberInterface
```

Comment:
Now that we deprecated the constants, does this split still make sense ?
Because we are using the deprecated constant here, and I'm not sure this is a good idea.